menu.txt

```diff
@@ -1,2 +1,3 @@
 Apple Pie
 Cheesecake
+Pudding
```

Comment:
Thanks for the changes. Looks like we are missing s after pudding.

Reply:
Thanks !! Will change to puddings